doc/api/util.markdown

```diff
@@ -335,6 +335,22 @@ util.isError({ name: 'Error', message: 'an error occurred' })
   // false
 ```
 
+Note that this method relies on `Object.prototype.toString()` behavior. It is
+possible to obtain an incorrect result when the `object` argument manipulates
+`@@toStringTag`.
+
+```js
+// This example requires the `--harmony-tostring` flag
```

Comment:
Aren't we deprecating all these functions with https://github.com/nodejs/node/pull/1301 (hopefully by v6.0)?